recipe/meta.yaml

```diff
@@ -11,30 +11,47 @@ source:
   sha256: "{{ sha256 }}"
 
 build:
+  # No mongodb on s390x
+  skip: True  # [py<34 or s390x]
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv --no-build-isolation
 
 requirements:
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
     - python
 
 test:
+  source_files:
+    - tests
   imports:
     - mockupdb
   requires:
-    - pymongo >=3
+    - pymongo >=3,<4  # [not (py>311 or (py>310 and osx and arm64) )]
+    # XXX even though we're not running the tests for py>311, we need
+    # *a* mongodb, which has a vendored bson, which mockupdb requires
+    # (for pip check).
+    - pymongo >=3     # [py>311 or (py>=310 and osx and arm64)]
```

Comment:
```suggestion
    - pymongo >=3     # [py>311 or (py>=310 and (osx and arm64))]
```

Reply:
Is there some clever maths thing here or is it just "we always bracket `osx and arm64`"?

Reply:
It's for readability only. Mainly, we bracket osx and arm64 if other AND/OR in the selector